robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
+
+Sitemap: https://www.deranged.dk/sitemap.xml
```

Comment:
Missing newline @ EOF

Reply:
Fixed in [bb0f63a](https://github.com/derangeddk/deranged.dk/pull/102/commits/bb0f63aad1c7f1cf170b2043c79c8d78389f4802)